### DISCUSSION TITLE
Welcome to data-science-practices-1 Discussions!

### DISCUSSION THREAD

**Discussion by wibeasley** [OuhscBbmc/data-science-practices-1#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mounisha-1212** [comment]
Hi Beasley.
Hope you are doing good.
I am Mounisha. and am from India.
currently am learning machine learning. hope this community will help me to practice more and learn more.
looking forward to join , work and learn new thing  with your community.
